aws/transit_gateway_test.go

```diff
@@ -283,12 +311,14 @@ func TestNukeTransitGatewayVpcAttachment(t *testing.T) {
 		},
 	})
 	require.NoError(t, err)
-	defer nukeAllTransitGatewayInstances(session, []*string{tgwVpcAttachment.TransitGatewayId})
-
+	
 	err = nukeAllTransitGatewayVpcAttachments(session, []*string{tgwVpcAttachment.TransitGatewayAttachmentId})
 	require.NoError(t, err)
-
+	
```

Comment:
Extraneous tab? might run `go fmt` on this.

Reply:
FIxed with `go fmt`